### ISSUE TITLE
Enable container logs and support failure on setup/startup scripts errors

### ISSUE THREAD

**Issue by iamazeem** [gvenzl/setup-oracle-free#6]
Hi,

Thank you for your work on this container and this action! :+1: 

Apparently, there's no way to check if the setup/startup scripts fail.
Also, there's nothing on the stdout/stderr under the container run group in the logs.

Currently, I'm using it like this and checking for errors in a subsequent step:

```yml
- name: Setup Oracle container
  uses: gvenzl/setup-oracle-free@v1
  with:
    container-runtime: docker
    app-user: app_user
    app-user-password: app_user_password
    setup-scripts: ${{ github.workspace }}/scripts

- name: Check Oracle container logs for errors
  run: |
    docker container ls
    docker container logs oracledb | tee -a /tmp/oracledb.log
    if grep 'ERROR' /tmp/oracledb.log >/dev/null; then
      echo "[ERR] Something went wrong!"
      echo "[ERR] See above logs for more details."
      exit 1
    fi
```

Would it be possible to provide some input parameter to enable these logs e.g. `enable-logs`?
Also, something like `fail-on-setup-errors` and/or `fail-on-startup-errors` may be added.

What do you think?
LMK if you need any help from side.

Thanks and regards!

**Comment by gvenzl** [comment]
Hey @iamazeem,

That's a great idea, I'll take a look.

Thanks,

**Comment by iamazeem** [comment]
@gvenzl: Thank you for considering this!

I recently wrote and published [setup-oracle-instant-client-action](https://github.com/iamazeem/setup-oracle-instant-client-action) to use OIC with `gvenzl/setup-oracle-free`.
I'm using `gvenzl/setup-oracle-free` in its [CI workflow](https://github.com/iamazeem/setup-oracle-instant-client-action/blob/main/.github/workflows/ci.yml) as well.

The [`docker container logs`](https://docs.docker.com/reference/cli/docker/container/logs/) command only returns the current batch of logs, not all of them.
I'm using a loop until all the logs are available like [this](https://github.com/iamazeem/setup-oracle-instant-client-action/blob/3781211817e1caf8a50a536bd310ac8ad3f6710e/.github/workflows/ci.yml#L53-L68):

```bash
docker container ls
while : ; do
  docker container logs oracledb | tee -a /tmp/oracledb.log
  if grep 'DATABASE IS READY TO USE!' /tmp/oracledb.log >/dev/null; then
    break
  fi
  sleep 1s
done
if grep 'ERROR' /tmp/oracledb.log >/dev/null; then
  echo "[ERR] Something went wrong!"
  echo "[ERR] See above logs for more details."
  exit 1
fi
```

You may need to account for this also.
Thank you!

**Comment by gvenzl** [comment]
Hey @iamazeem, 

The fix is now live with `v1.1.0` (or automatically via `v1`).
Whenever the container fails or a timeout occurs, the logs are printed.
This includes failures in setup or startup scripts that terminate the container.

Thanks a lot for this suggestion, I think it will help a lot of users.

Btw,
